tools/make/lib/install/modules.mk

```diff
@@ -16,25 +16,8 @@
 # limitations under the License.
 #/
 
-# VARIABLES #
-
-# Define a path to a utility for printing the list of node module dependencies to install:
-print_npm_install_deps := $(TOOLS_PKGS_DIR)/scripts/print_npm_install_deps
-
-# Define the file path for storing the list of node module dependencies to install:
-npm_install_deps_file := $(CONFIG_DIR)/npm/deps.txt
```

Comment:
This file is actually a tracked file. Each time the root `package.json` file changes and/or a dev clones, `make install_node_modules` clears the existing `node_modules` directory and reinstalls. This then updates the `deps.txt` file (a tracked file) to have a static record of the latest deps the repo depends on. So eliminating this recipe and no longer updating this file is not desired.